recipes/finmind/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "finmind" %}
+{% set version = "1.0.84" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FinMind-{{ version }}.tar.gz
+  sha256: f19bc2a7c9698ea80c4a78d2d3eb27daad01c8bec896cc34da9ff8f74a413644
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```

It seems that this project depends on ``pandas``, ``lxml``, ``requests``, ``numpy``, ``keras``, ``matplotlib``, ``scikit-learn``. Am I right? If so, could you please add them as well?
```suggestion
    - python
    - pandas
    - lxml
    - requests
    - numpy
    - keras
    - matplotlib
    - scikit-learn
```

---

but when i pip install FinMind,
pands and matplotlib are not finmind's dependencies.

i think i should keep it that way.

---

its a package to import some stock datasets as json file.
we dont need to crawl the website.
https://github.com/FinMind/FinMind#what-is-it


https://github.com/FinMind/FinMind#translation-%E4%B8%AD%E8%8B%B1%E5%B0%8D%E7%85%A7
of course we could use pandas to process json dataset from finmind
but we dont really NEED pandas.